docs/developer_guide/testing.rst

```diff
@@ -1,29 +1,58 @@
 Testing
 =======
 
-Mantid Imaging uses unit tests, static analysis and GUI approval testing
+Mantid Imaging uses a combination of unit tests, static analysis, and GUI approval testing to maintain the stability and quality of the codebase.
 
-The test suite can be run using our makefile target. Note this does not include the system and screenshot tests::
+The basic test suite, which includes unit tests and static analysis, can be run using our Makefile target. Note that this command does not include the system and screenshot tests::
 
     make check
 
-On Windows, you may need to install a Windows version of GNU Make.::
+On Windows, you may need to install a Windows version of GNU Make::
 
     mamba install make
 
-Tests are run automatically on pull requests on GitHub using actions, but should also be run during development.
-The full suite of unit tests can run very slowly on a Windows machine. We recommend selecting only relevant tests to run locally when developing on Windows.
+Tests are run automatically on pull requests via GitHub Actions, but should also be run locally during development to catch issues early. The full suite of unit tests can run very slowly on a Windows machine, so we recommend selecting only the relevant tests to run locally during development.
+
+For more targeted testing, Mantid Imaging provides several specific commands:
+
+- :code:`make test` : Runs the basic unit test suite to ensure that all unit tests pass. This is ideal after making code changes to confirm that nothing is broken.
+- :code:`make test-verbose` : Executes the unit test suite with verbose output, providing detailed information about each test. Useful for gaining more insight when debugging test failures.
+- :code:`make test-system` : Runs the system tests, which are typically end-to-end tests that validate the overall functionality of the system. This is particularly useful for confirming that the entire system functions correctly after major updates.
+
+Refer to the :file:`Makefile` for additional options or to modify these commands for specific scenarios.
+
+Running Specific Tests
+----------------------
+
+Mantid Imaging allows you to focus on specific test cases or workflows by running individual test modules or selecting tests based on keywords.
+
+To run a specific test module, you can specify its path using the following command:
+
+    python -m unittest path.to.your_test_module
```

Comment:
Need a double colon `::` to make a code box.

Also, lets stick to pytest commands for consistancy

`pytest path/to/test_file.py`